src/actions/index.js

```diff
@@ -0,0 +1,9 @@
+export const emailChanged = text => ({
+	type: 'EMAIL_CHANGED',
```

Comment:
By convention, actions are usually named in an active tense, like ‘changeEmail’ or ‘changePassword’